pkg/kubelet/nodeconfig/controller.go

```diff
@@ -0,0 +1,270 @@
+/*
```

Comment:
All logs from the controller pass through a shim that prepends `nodeconfig controller: ` to the log message. See `log.go`. This is cleaner than having to add `nodecontroller_` to every filename.